src/messaging/ExchangeContext.h

```diff
@@ -214,6 +214,47 @@ class DLL_EXPORT ExchangeContext : public ReliableMessageContext,
      */
     bool IsSendExpected() const { return mFlags.Has(Flags::kFlagWillSendMessage); }
 
+    /**
+     * Notifies the exchange that our consummer is going to send a message where it knows the peer (receiver) should be active
```

Comment:
If we do the "we are sending a response" thing above, under what conditions would this API be useful?

Reply:
resolving comment since it is no longer applicable